app/views/emails/email.erb

```diff
@@ -0,0 +1,13 @@
+<%- model_class = Email -%>
+<div class="page-header">
+</div>
+<div>
+  <div class="email-form col-md-8">
+    <%= render partial: 'emails/email_form', locals: { email: @email } %>
+  </div>
+  <div class="email-templates col-md-4">
+    <% if params[:status] %>
+      <%= render partial: 'emails/templates', locals: { template: @templates } %>
```

Comment:
@tobiduer wants a third status for templates 'others' that are displayed if no other status is given, therefore will be changed